lib/cli/interactive-setup/deploy.js

```diff
@@ -53,7 +53,14 @@ module.exports = {
     return false;
   },
   async run(context) {
-    const { configuration, configurationFilename, serviceDir, stepHistory } = context;
+    const { configuration, configurationFilename, serviceDir, stepHistory, history } = context;
+    if (configuration.org && configuration.app && !history.has('dashboardSetOrg')) {
```

Comment:
Just to be sure, the new message will not show if a new project was created? (because I guess the only case `dashboardSetOrg` doesn't run is when org & app where set up, so the project must have existed before) Just making sure

Reply:
It will only show for situations where there's an existing project that already has dasboard setup (org and app configured service config).